editors/code/src/commands/server_version.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from 'vscode';
+import { ensureServerBinary } from '../installation/server';
+import { Ctx, Cmd } from '../ctx';
+import { spawnSync } from 'child_process';
+
+export function serverVersion(ctx: Ctx): Cmd {
+    return async () => {
+        const binaryPath = await ensureServerBinary(ctx.config.serverSource);
+
+        if (binaryPath == null) {
+            throw new Error(
+                "Rust Analyzer Language Server is not available. " +
+                "Please, ensure its [proper installation](https://rust-analyzer.github.io/manual.html#installation)."
+            );
+        }
+
+        const res = spawnSync(binaryPath, ["--version"]);
+        const version = res.output?.filter(x => x !== null).map(String).join(" ");
```

Comment:
Since `--version` is printed to `stdout`, there is no need to iterate over all standard fstreams.
```suggestion
        const version = spawnSync(binaryPath, ["--version"], { encoding: "utf8" }).stdout;
```

Reply:
Fixed.